modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro

```diff
@@ -5,4 +5,89 @@ definition {
 		Click(locator1 = "PRMMarketing#APPLY_FOR_FUNDS");
 	}
 
+	@summary = "Verify the activity fills made by the user"
```

Comment:
Hi @marinaasant0s, could you please update this summary to: "View the activity information on MDF Review Page"?